diff --git a/third-party-podspecs/ReactNativeDependencies.podspec b/third-party-podspecs/ReactNativeDependencies.podspec
index 32ebe58e05f46f9ccbbd582424aee89b7a609830..9c9060f7babba43bc6969100bc83b8331c031ff2 100644
--- a/third-party-podspecs/ReactNativeDependencies.podspec
+++ b/third-party-podspecs/ReactNativeDependencies.podspec
@@ -64,8 +64,8 @@ Pod::Spec.new do |spec|
 
     cp -R "$HEADERS_PATH/" Headers
     mkdir -p framework/packages/react-native
-    cp -R "$XCFRAMEWORK_PATH/.." framework/packages/react-native/
-    find "$XCFRAMEWORK_PATH/.." -type f -exec rm {} +
+    cp -R "$XCFRAMEWORK_PATH/../." framework/packages/react-native/
+    find "$XCFRAMEWORK_PATH/.." -type f -exec rm {} +
     find "$CURRENT_PATH" -type d -empty -delete
   CMD
 
